Description: Remove non-existent argument from help
Author: Sasha Finkelstein <fnkl.kernel@gmail.com>
Applied-Upstream: db7719d110d99e0ef8fb23a99f1ff9f6ff4bcb8d
Reviewed-by: NoisyCoil <noisycoil@tutanota.com>
Last-Update: 2025-04-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/main.rs
+++ b/src/main.rs
@@ -67,7 +67,6 @@
             clap::Command::new("sync")
                 .about("Sync Bluetooth device information from nvram")
                 .arg(clap::arg!(-c --config [CONFIG] "Bluez config path."))
-                .arg(clap::Arg::new("variable").multiple_values(true)),
         )
         .subcommand(
             clap::Command::new("dump").about("Dump binary Bluetooth device info from nvram"),
